### PR TITLE
fix: [CDS-73422]: modified ux for pipeline exec summary

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

_**Screenshots before:**_
Screenshot 2023-07-18 at 10 22 18 AM

Screenshot 2023-07-18 at 10 22 33 AM

Screenshot 2023-07-18 at 10 22 48 AM

### CODE REVIEW

**Comment by sahithikolichala** [comment]
modified ux 

Screenshot 2023-07-17 at 3 35 35 PM
.com/harness/harness-core-ui/assets/57
Screenshot 2023-07-17 at 3 36 01 PM
372986/1ee526ec-5111-41da-a720-ebb5421f7811">

**Comment by sahithikolichala** [comment]
Screenshot 2023-07-17 at 3 36 25 PM

**Comment by sahithikolichala** [comment]
retrigger cypress-cv

**Comment by sahithikolichala** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/pages/execution/ExecutionPipelineView/ExecutionGraphView/ExecutionStageDetailsHeader/ExecutionStageDetailsHeader.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export function ExecutionStageDetailsHeader(): React.ReactElement {
         <div className={css.lhs} data-has-sibling={Boolean(stage && stageDetail?.component)}>
           <div className={css.stageTop}>
             <Text margin={{ bottom: 'small' }} font={{ variation: FontVariation.H6 }} lineClamp={1}>
-              {stage?.name}
+              {stage?.name}sdfdsfds
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution/ExecutionPipelineView/ExecutionGraphView/ExecutionStageDetailsHeader/ExecutionStageDetailsHeader.tsx:220]: remove

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx [hunk 2]
```diff
@@ -198,9 +203,18 @@ export default function ExecutionMetadata(): React.ReactElement {
...
+  const additionalAttrs =
+    cIInfo || cdInfo
+      ? {
+          'data-has-sibling': true
+        }
+      : {}
   return (
     <div className={css.main}>
-      <div className={css.metaContainer}>
+      <div className={`${classList}`} {...additionalAttrs}>
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx:217]: use `classnames`

**Review by rameshx — CHANGES_REQUESTED** [review]

**Comment by rameshx** [comment]
@sahithikolichala Could you add before and after screenshots in the description?

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Comment by sahithikolichala** [comment]
@rameshx  added the screenshots before and after

**Review by rameshx — APPROVED** [review]

**Comment by sahithikolichala** [comment]
retrigger cypress-cv

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx [hunk 3]
```diff
@@ -146,6 +150,9 @@ export default function ExecutionMetadata(): React.ReactElement {
   const ciData = factory.getSummary(StageType.BUILD)
   const cdData = factory.getSummary(StageType.DEPLOY)
 
+  const cIInfo = HAS_CI && ciData
```
Comment (srdjan-harness) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx:153]: to keep camel sase style maybe is better to have `ciInfo`

File: src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx [hunk 4]
```diff
@@ -198,9 +205,20 @@ export default function ExecutionMetadata(): React.ReactElement {
     ) : null
     return visible
   }
+
+  const additionalAttrs =
+    cIInfo || cdInfo
```
Comment (srdjan-harness) [inline src/modules/70-pipeline/pages/execution/ExecutionLandingPage/ExecutionMetadata/ExecutionMetadata.tsx:210]: i would rather add var for this ciOrCdInfo or cicdInfo below `const cdInfo = HAS_CD && cdData` as condition is used in three places.

**Review by srdjan-harness — CHANGES_REQUESTED** [review]

**Review by srdjan-harness — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Pull request merged**